app/assets/javascripts/discourse/tests/acceptance/fast-edit-test.js

```diff
@@ -88,7 +88,7 @@ acceptance("Fast Edit", function (needs) {
     await visit("/t/internationalization-localization/280");
 
     query("#post_2 .cooked").append(`That’s what she said!`);
-    const textNode = query("#post_2 .cooked").childNodes[2];
+    const textNode = query("#post_2 .cooked").childNodes[3];
```

Comment:
Curious, why does the index here need to be changed? The new element is appended, so shouldn't node 2 here still refer to the text in question?

Reply:
It's because my change adds a `div` element to the cooked content.

This moves the element the test is appending by one position.